src/query/functions/src/srfs/mod.rs

```diff
@@ -241,6 +253,197 @@ pub fn register(registry: &mut FunctionRegistry) {
             },
         }))
     });
+
+    registry.properties.insert(
```

Comment:
It's better to move the logic code from `mod.rs` to a new file like `register.rs`

Reply:
Good idea, too much logic indeed, I'll do a bit of splitting up.